py3status/modules/whatismyip.py

```diff
@@ -94,9 +95,17 @@ class Meta:
         }
 
     def post_config_hook(self):
+        self.headers = {'User-Agent': UA}
+
         if self.expected is None:
             self.expected = {}
 
+        # Alert about negative_cache_timeout
+        if self.negative_cache_timeout < 60:
+            self.py3.error(
+                'warning, negative_cache_timeout should be > 60 to prevent ' +
+                'from being blacklisted by API')
```

Comment:
This doesn't account for the fact that I might be using my own self-hosted API that allows me to make requests much more often 😉